ksqldb-functional-tests/src/test/resources/query-validation-tests/suppress.json

```diff
@@ -15,10 +92,222 @@
+      ]
+    },
+    {
+      "name": "should support final results for tumbling windows",
+      "statements": [
+        "CREATE STREAM INPUT (ID STRING KEY, COL1 STRING) WITH (kafka_topic='input_topic',value_format='JSON');",
+        "CREATE TABLE OUTPUT AS SELECT ID, COUNT(*) as COUNT FROM INPUT WINDOW TUMBLING (SIZE 2 MILLISECONDS, GRACE PERIOD 1 MILLISECONDS) GROUP BY ID EMIT FINAL;"
+      ],
+      "inputs": [
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 1},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 2},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 1},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 5}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 4},"window": {"start": 0, "end": 2, "type": "time"},"timestamp": 1},
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 1},"window": {"start": 2, "end": 4, "type": "time"},"timestamp": 2}
+      ]
+    },
+    {
+      "name": "should support final results for tumbling windows with large jump",
+      "statements": [
+        "CREATE STREAM INPUT (ID STRING KEY, COL1 STRING) WITH (kafka_topic='input_topic',value_format='JSON');",
+        "CREATE TABLE OUTPUT AS SELECT ID, COUNT(*) as COUNT FROM INPUT WINDOW TUMBLING (SIZE 2 MILLISECONDS, GRACE PERIOD 2 MILLISECONDS) GROUP BY ID EMIT FINAL;"
+      ],
+      "inputs": [
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 1},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 2},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 3},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 4},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 30}
+
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 4},"window": {"start": 0, "end": 2, "type": "time"},"timestamp": 1},
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 2},"window": {"start": 2, "end": 4, "type": "time"},"timestamp": 3},
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 1},"window": {"start": 4, "end": 6, "type": "time"},"timestamp": 4}
```

Comment:
what if we made the third window start at a value in the middle instead of consecutive 0 -> 2 -> 4, like 10?